app.rb

```diff
@@ -0,0 +1,46 @@
+require 'sinatra/base'
+require 'sinatra/flash'
+require './lib/peep'
+require './lib/user'
+
+class Chitter < Sinatra::Base
+  register Sinatra::Flash
+  enable :sessions
+
+  get '/' do
+    @user = User.find(session[:user_id])
+    erb :index
+  end
+
+  post '/' do
```

Comment:
I'm not clear what the purpose of this route it - same for the post /users/new route on line 30